girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -445,6 +447,55 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
                 removed += 1
         return removed
 
+    @access.user(scope=TokenScope.DATA_WRITE)
+    @autoDescribeRoute(
+        Description('Create large images for all items within a folder.')
+        .notes('Does not work for items with multiple files.')
+        .modelParam('id', 'The ID of the folder.', model=Folder, level=AccessType.WRITE, required=True)
+        .param('createJobs','Whether a job should be created for each large image.', required=False, 
+            default=False, dataType='boolean')
+        .param('localJobs', 'Whether a the jobs created should be local.', required=False, 
+            default=False, dataType='boolean')
+        .param('recurse', 'Whether child folders should be recursed', required=False, default=False, 
+            dataType='boolean')
+        .errorResponse('ID was invalid.')
+        .errorResponse('Write access was denied for the folder.', 403)
+    )
+    def createLargeImages(self, folder, params):
+        user=self.getCurrentUser()
+        createJobs = params.get('createJobs')
+        if createJobs:
+            createJobs = 'always' or True
+        self.createImagesRecurseOption(folder=folder, createJobs=createJobs, user=user, 
+                                recurse=params.get('recurse'), localJobs=params.get('localJobs'))
+                    
+    def createImagesRecurseOption(self, folder, createJobs, user, recurse, localJobs):
+        count = {'large images created': 0}
+        if recurse:
+            for childFolder in Folder().childFolders(parent=folder, parentType='folder'):
+                self.createImagesRecurseOption(folder=childFolder, createJobs=createJobs, user=user, 
+                                        recurse=recurse, localJobs=localJobs)
+        for item in Folder().childItems(folder=folder):
```

Comment:
Does this call to `childItems` also need `limit=0`? And the call to `childFolders` a few lines up?

Reply:
Removed that [here](https://github.com/girder/large_image/pull/1572/commits/ef19b736766a8f3410552291b78805fc62bc4563).